Got the following error in Travis CI build(build 29825418), not sure why.

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.7:deploy (default-deploy) on project rescu: Failed to deploy artifacts: Could not transfer artifact com.github.mmazi:rescu:jar:1.7.0-20140713.140144-2 from/to sonatype-nexus-snapshots (http://oss.sonatype.org/content/repositories/snapshots): Failed to transfer file: https://oss.sonatype.org/content/repositories/snapshots/com/github/mmazi/rescu/1.7.0-SNAPSHOT/rescu-1.7.0-20140713.140144-2.jar. Return code is: 401, ReasonPhrase: Unauthorized. -> [Help 1]

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,7 +2,13 @@ language: java
 jdk:
   - openjdk6
 before_install: "git clone -b travis `git config --get remote.origin.url` target/travis"
-script: "mvn deploy --settings target/travis/settings.xml"
+script: "
+if [ ${TRAVIS_PULL_REQUEST} = 'false' ];
+then
+  mvn deploy --settings target/travis/settings.xml;
+else
+  mvn clean verify --settings target/travis/settings.xml;
+fi"
 
 # whitelist
 branches:

